### DISCUSSION TITLE
Synth preset browser

### DISCUSSION THREAD

**Discussion by Dubfella** [SynthstromAudible/DelugeFirmware#647] [Ideas]
It would be really cool to add a preset browser for the synthesizer. For example, I have several folders with various patches: Original synthstrom patches, Boards of Deluge, and a folder with my patches. If I could navigate through these patch folders in the browser it would be very convenient. 
What you think about idea?
Key combinations: double tap on synt key, or audition key + synth key, or synt key + rotate browse

**Comment by ok-reza** [comment]
how is this different than the current preset browser? 'load + synth' and you can scroll through presets, if you find a folder you want to explore you enter it via 'load' or pressing select and can explore that folder, hit 'back' to exit the directory
